### PR TITLE
Feature/COMM-2231  login logic

### PR DESCRIPTION
**Pull request by ivanzusko**

### CODE REVIEW

File: src/modules/pages/LoginPage/LoginPage.js [hunk 1]
```diff
@@ -1,16 +1,7 @@
...
 
-const LoginPage = () => (
-    <div>
-        <Section>
-            <Title>Login page</Title>
-        </Section>
-        <LoginContainer />
-    </div>
-);
+const LoginPage = () => <LoginContainer />;
```
Comment (ivanzusko) [inline src/modules/pages/LoginPage/LoginPage.js]: Sure, it was possible to omit this file by passing straight to the root, but I left it just for consistency and for the case if something except of the form will be needed on this page

File: src/modules/Layout/Row/Row.scss [hunk 2]
```diff
@@ -1,5 +1,20 @@
 .j-row {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
+    display: flex;
+    width: 100%;
```
Comment (Vxplus) [inline src/modules/Layout/Row/Row.scss]: can you try a combination of `flex-grow` and, `flex-shrink` and see if you still need the 100%

File: src/modules/pages/LoginPage/LoginPage.js (hunk 1 above)
Comment (Vxplus) [inline src/modules/pages/LoginPage/LoginPage.js]: I'm for consistency but keep consistency in being consistent

File: src/modules/Layout/Row/Row.scss (hunk 2 above)
Comment (ivanzusko) [inline src/modules/Layout/Row/Row.scss]: @Vxplus yeap, you are right, thanks! By default `flex-grow` is one, what is exactly what I need.

**Pull request merged**